components/artwork_masonry/view.coffee

```diff
@@ -31,15 +33,26 @@ module.exports = class ArtworkMasonryView extends Backbone.View
       .check @artworks.map ({ id }) -> id
 
   reset: (artworks) ->
-    invoke @subViews, 'remove'
+    _.invoke @subViews, 'remove'
```

Comment:
This should still be `invoke` unless you remove the destructured assignment above.